ConsoleApp/Food.cs

```diff
@@ -0,0 +1,23 @@
+namespace ConsoleApp;
+    class Food
+    {
+        int x;

```

Comment:
code convention violated, should be
```suggestion
        private int _x;

```